cirq-google/cirq_google/engine/simulated_local_processor.py

```diff
@@ -37,7 +37,7 @@
     'type.googleapis.com/cirq.google.api.v2.BatchProgram',
 ]
 
-GATE_SET_VALIDATOR_TYPE = Callable[
+PROGRAM_VALIDATOR_TYPE = Callable[
```

Comment:
This type is duplicated in `engine_validator.py`. Can this be imported from there instead (or some other common location)?

Reply:
Thanks for the catch, removed duplicates.